frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -171,5 +172,10 @@ export const toApiProject = (project: IProject) => {
     exemptionRationale: project.exemptionRationale,
     exemptionRequested: project.exemptionRequested,
   };
+  // convert all empty strings (required by formik) to undefined
+  Object.keys(apiProject).forEach(key => {
```

Comment:
Of course, going back from app -> api we can't/don't want to write a whole bunch of empty strings back, so clean those up before submission.